lib/Enum/ScrollingDirection.lua

```diff
@@ -0,0 +1,5 @@
+return {
+	X = 1,
+	Y = 2,
```

Comment:
`ScrollingDirection` has space for a `Z` value I see 😄

Reply:
haha looks like they're doing the 1<<0, 1<<1< 1<<2 pattern in the cpp file